web-src/src/helpers/FeatureFlagsHelper.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import featureFlags from '@adobe/exc-app/featureflags';
+
+const PROJECT_ID = 'aem-generate-variations';
+
+export async function getFeatureFlags() {
+  const flags = await featureFlags.get([PROJECT_ID]);
+  return flags[PROJECT_ID];
```

Comment:
Does the contract of `@adobe/exc-app/featureflags` really asks you twice for the project id? Appears redudant. I see it being used in L17 and L18.

Reply:
It's strange but it works. Just tried it myself.